m03_acos/create_acos_exercises.rst

```diff
@@ -0,0 +1,454 @@
+    :linenos:
+
+    {
+      "draggables":{
+        "source":{
+          "content":"<img src=\"/static/draganddrop-data-management/src_folder.png\" alt=\"folder\">",
+          "reuse":false,
+          "feedback":{
+            "folder":"Correct!, this folder contains the source files.",
+            "DEFAULT":"This folder only contain the source files."
+          }
+        },
+        "date_folder":{
+          "content":"<img src=\"/static/draganddrop-data-management/date_folder.png\" alt=\"folder\">",
+          "reuse":false,
+          "feedback":{
+            "folder":"Correct!, this folder contains information organised by date",
+            "DEFAULT":"This folder contains information organised by date."
+          }
+        },
+        "experiment":{
+          "content":"<img src=\"/static/draganddrop-data-management/exp_folder.png\" alt=\"folder\">",
+          "reuse":false,
+          "feedback":{
+            "folder":"Correct!, this folder contain the results of different experiments",
+            "DEFAULT":"This folder contains the results of different experiments."
+          }
+        },
+        "file":{
+          "content":"<img src=\"/static/draganddrop-data-management/file.png\" alt=\"folder\">",
+          "reuse":true,
+          "feedback":{
+            "folder":"Correct!, this is a group of files",
+            "DEFAULT":"This is a list of different files."
+          }
+        },
+        "readme":{
+          "content":"<img src=\"/static/draganddrop-data-management/readme.png\" alt=\"folder\">",
+          "reuse":true,
+          "feedback":{
+            "folder":"Correct!, This is a readme file",
+            "DEFAULT":"This is a readme file that contains meta data.."
+          }
+        }
+      },
+      "droppables":{
+        "source":{
+          "correct":[
+            "source"
+          ]
+        },
+        "date_folders":{
+          "correct":[
+            "date_folder"
+          ]
+        },
+        "experiments":{
+          "correct":[
+            "experiment"
+          ]
+        },
+        "files":{
+          "correct":[
+            "file"
+          ]
+        },
+        "readme":{
+          "correct":[
+            "readme"
+          ]
+        }
+      }
+    }
+  
+Once your Acos exercise is ready, and the exercise files have been created, you will have two choices for testing your code locally in the browser. :ref:`The first option <local-test>` is to test the exercise with a copy of the Acos-server in your computer. :ref:`The second option <docker-test>` is to test the exercise by using a pre-installed docker container which is downloaded after you have compiled an A+ course for the first time. 
+
+.. warning:: Before proceeding to test your Acos exercise, you should :ref:`configure the localhost <acos-docker-mac-windows>` in your computer. Therefore, you should verify that your host file have the ``127.0.0.1 acos`` configuration.
```

Comment:
See comment about warnings on lines 370-372. If one doesn't configure the localhost, does something bad happen? If not, a title such as "Note" might be more applicable than "Warning". Of course that might be difficult to implement with the yellow box, I don't know.

Reply:
So, this one should be also something like `important` or `note`?

Reply:
Yep!